README.md

```diff
@@ -2,11 +2,7 @@
 
```

Comment:
- [ ] The link for the license on your repository appears to be broken. Kindly add a license to the repository, you can follow this [link](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository) to see how to do so. Having functional links on the README is very important to give a professional appearance to your repository